### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by sims1710**
I created a theme which focuses on pastel pink, hope you find okay and can add it to your themes!

### CODE REVIEW

File: themes/index.js [hunk 1]
```diff
@@ -430,6 +430,12 @@ export const themes = {
     border_color: "ffffff",
     bg_color: "000000",
   },
+  pastel_pink: {
+    title_color: "EC7E9B",
+    text_color: "A6A6A6",
+    icon_color: "FFD9DD",
+    border_color: "e4e2e2",
+    bg_color: "fffefe"
```
Comment (rickstaa) [inline themes/index.js:438]: I think you have a missing comma here.

Comment (sims1710) [reply themes/index.js]: Oh I see, thanks a lot for pointing out!

**Review by qwerty541 — COMMENTED** [review]
@sims1710 I have resolved JSON syntax and contrast check errors on your pull request. Now it will no be automatically closed, but card looks a little different. Feel free to revert changes.

**Comment by anuraghazra** [comment]
We are currently pausing addition of new themes. If this theme is exclusively for your personal use, then instead of adding it to our theme collection, you can use card [customization options](https://github.com/anuraghazra/github-readme-stats#customization).

**Pull request closed without merging**